cli/Cargo.toml

```diff
@@ -34,6 +34,7 @@ byteorder = "1.3.4"
 clap = "2.33.1"
 dissimilar = "1.0.2"
 dlopen = "0.1.8"
+encoding = "0.2"
```

Comment:
The `encoding` crate has not been actively maintained since 2017, I think the [encoding_rs](https://crates.io/crates/encoding_rs) crate could be a better choice.

Reply:
Alright, I'm looking into that, but as far as I can see the encoding_rs doesn't provide a convenient, high level way of doing this - its usage requires much more code to handle feeding the buffers etc., see: https://github.com/hsivonen/recode_rs/blob/master/src/main.rs

Reply:
I don't think so? You can take a look at `reqwest`'s usage: https://github.com/seanmonstar/reqwest/blob/a800202384484f60679cdd884b08ba1e320212e3/src/async_impl/response.rs#L182-L194

Reply:
Oh wow, this is indeed much simpler! 
Thank you very much for the reference